src/customize-pop/gnome-tweaks-extensions.md

```diff
@@ -0,0 +1,202 @@
+# GNOME Tweaks & Extensions
+
+GNOME Tweaks and GNOME Shell Extensions provide additional customization beyond the `Settings` menu.
+
+---
+
+## GNOME Tweaks
+
+GNOME Tweaks is an optional software package allowing users to further customize the appearance and behavior of Pop!\_OS. GNOME Tweaks features include changing power settings, managing startup applications, enabling desktop icons, and changing fonts.
+
+### Install GNOME Tweaks
+
+Launch the Pop!\_Shop from the Dock and search “gnome tweaks”, click `Install`.
+
+![Install Tweaks](/images/gnome-tweaks-extensions/install-tweaks.png)
+
+## GNOME Tweaks Settings
+
+### General
+
+![General Settings](/images/gnome-tweaks-extensions/general-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Suspend when laptop lid is closed | Place the system into a suspended state when the laptop lid is closed. |
+| Over-Amplification | Allow the volume to be increased above 100%. |
+
+### Appearance
+
+`Appearance` settings modify the theme, background, and lock screen elements of the desktop environment.
+
+![Appearance Tweaks](/images/gnome-tweaks-extensions/appearance-tweaks.png)
+
+#### Theme Options
+
+| Option | Function |
+|--------|----------|
+| Applications | Select the theme that will be applied to application windows. |
+| Cursor | Select a cursor design. |
+| Icons | Choose an icon set that displays custom icons for system tools and applications. |
+| Shell | Select or upload a shell theme. Using this option requires enabling `User Themes` in GNOME Extensions. |
```

Comment:
I'll create a procedure as a sub-chapter showing how to enable and add a user theme. I felt this may be out of scope, but it may help users get back to the default Pop!_OS theme should they ever modify their system and something breaks.

Reply:
I'll add this and we can further discuss how in-depth this section should be. Modifying the shell and desktop GUI is the most visibly impactful customization, and highlights the flexibility offered by Linux to new users who found their way to Pop!_OS as a first distro. I'll take my best stab at the Terminal instructions but may need your help.